### DISCUSSION TITLE
How to Hide Tabline in Dashboard

### DISCUSSION THREAD

**Discussion by Ducky6944** [NvChad/NvChad#628] [Q&A] [answered]
When the dashboard loads after starting nvim it shows the tabline. Trying to disable this with an autocmd in my autochad_cmds.lua doesn't seem to work for the initial display of dashboard, but it does seem to work for subsequent. Not sure if I should bypass the custom configs or if there is a better way. 

autochad_cmds.lua
```VIML
vim.api.nvim_exec([[ autocmd FileType dashboard set showtabline=0 | autocmd WinLeave <buffer> set showtabline=2 ]], false)
```

I was able to work around this by creating a new bufferline config and changing `always_show_bufferline = false,` from `true`.  I'm not sure what other effects this might have though. Insight on the autocmd solution would still be appreciated.

**Comment by siduck** [comment, marked as answer]
can you try this 

```
vim.cmd [[
autocmd FileType dashboard set showtabline=0 | autocmd WinLeave <buffer> set showtabline=2
]]
```

**Reply by Ducky6944** [reply]
I need to look into the different APIs it seems. Thanks again!

**Reply by siduck** [reply]
> I need to look into the different APIs it seems. Thanks again!

that was provided in dashboard's readme but it doesnt work for me :(

**Reply by siduck** [reply]
https://user-images.githubusercontent.com/59060246/144969764-a67483e8-f76a-4f82-9be7-675b2580d02d.mp4
 it works when i go fullscreen

**Reply by siduck** [reply]
btw do checkout https://github.com/startup-nvim/startup.nvim , it disables tabline by default and a lil faster too i must say. I'll see if it works out well , if it does then i'll replace dashboard-nvim with it.

**Reply by Ducky6944** [reply]
Hey thanks startup looks really nice. 

I've got another dumb question here, but when trying to setting fillchars with `vim.opt.fillchars = { fold = " " }` nothing appears to get set. I do see eob set in core/options and setting fold in there does work, so I believe I've set it properly. It also works setting it the command editor. I'm trying to get something like [this](https://github.com/AdamWagner/stackline/issues/42#issuecomment-696817874) and all the other commands work so far.

**Reply by siduck** [reply]
Im not sure how folding works , I dont use it but looks interesting! you could ask it in our discord server , maybe some knowledgeable in this matter might help you :0

**Reply by Ducky6944** [reply]
I think for some reason setting options that are in core.options but isn't actually set to be changed with the M.options doesn't work. Maybe it has something to do with the table merge in core.utils.
Changing `opt.fillchars = { eob = " " }` to `opt.fillchars = options.fillchars` then setting `fillchars = { eob = " ", fold = " " }` in chadrc  resolved the issue. It could probably be forced load with an autocmd too, but that's not ideal.

**Reply by siduck** [reply]
oh , I forgot to add an option for that in default_config.lua so you were facing that issue!

**Reply by siduck** [reply]
added it https://github.com/NvChad/NvChad/commit/01babafd7286e42199bf69d6f6a70c6958c73940

**Reply by Ducky6944** [reply]
You're a true NVChad thanks!
